core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -593,30 +522,28 @@ private DefaultFunctionResolver date_sub() {
                 .toArray(SerializableFunction<?, ?>[]::new));
   }
 
-  /** DAY(STRING/DATE/DATETIME/TIMESTAMP). return the day of the month (1-31). */
+  /** DAY(STRING/DATE/TIMESTAMP). return the day of the month (1-31). */
   private DefaultFunctionResolver day() {
     return define(
         BuiltinFunctionName.DAY.getName(),
         impl(nullMissingHandling(DateTimeFunction::exprDayOfMonth), INTEGER, DATE),
-        impl(nullMissingHandling(DateTimeFunction::exprDayOfMonth), INTEGER, DATETIME),
         impl(nullMissingHandling(DateTimeFunction::exprDayOfMonth), INTEGER, TIMESTAMP),
         impl(nullMissingHandling(DateTimeFunction::exprDayOfMonth), INTEGER, STRING));
   }
 
   /**
-   * DAYNAME(STRING/DATE/DATETIME/TIMESTAMP). return the name of the weekday for date, including
-   * Monday, Tuesday, Wednesday, Thursday, Friday, Saturday and Sunday.
+   * DAYNAME(STRING/DATE/TIMESTAMP). return the name of the weekday for date, including Monday,
+   * Tuesday, Wednesday, Thursday, Friday, Saturday and Sunday.
```

Comment:
Can you add a `<br>` before `Monday`  to force it onto the same line as the other dates.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/f872540c9d37618958c34457bf6c8bb884c7e559